src/assets/images/close.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should probably do some research into what we can/should remove from these svg files